emr-etl-runner/Makefile

```diff
@@ -0,0 +1,84 @@
+.PHONY: all help build run builddocker rundocker kill rm-image rm clean enter logs
+
+all: help
```

Comment:
removed, but I'm not certain why, they are a best practice in makefiles, that is to detail which recipes do not produce a file of the same name, and the all recipe is called if someone just calls `make` with no arguments.